src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/Pal/Unix/DecryptorPalOpenSsl.Decode.cs

```diff
@@ -101,12 +101,8 @@ internal sealed partial class DecryptorPalOpenSsl : DecryptorPal
                 innerSequenceReader.SkipValue();
             }
 
-            // Skip recipientInfos as these are retrieved with OpenSSL
-            const byte constructedSequence = (byte)DerSequenceReader.DerTag.Set | DerSequenceReader.ConstructedFlag;
-            Debug.Assert(innerSequenceReader.PeekTag() == constructedSequence);
-            innerSequenceReader.SkipValue();
-
-            contentEncryptionAlgorithm = innerSequenceReader.ReadEncryptionAlgorithm();
+            recipientInfos = GetRecipients(innerSequenceReader);
```

Comment:
Technically a recipient is a person, and a RecipientInfo is what you're reading.

I'm also confused about the style.  The existing ones are extension methods named Read.  The new one is a non-extension method named Get.  (Read vs Get is what I care about more)